setup/ansible/rhel8.yml

```diff
@@ -0,0 +1,161 @@
+# rhel8.yml
+# Ansible Playbook to deploy OBS according to 
+# https://github.com/OSEC-pl/obs/blob/master/setup/rhel8.txt
+#
+# Copyright 2020 Steven Bonneville 
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Configure OBS on RHEL 8
+  hosts: all
+  become: yes
+
+  vars:
+    rhel8_build_tools:
+      - git
+      - vim
+      - tmux
+      - "@Development Tools"
+      - kernel-devel
+      - kernel-headers
+      - elfutils-libelf-devel
+      - cmake
+      - qt5-qtbase-devel
+
+  tasks:
+    - name: Kernel is up to date
+      yum:
+        name: kernel
+        state: latest
+      register: kernel_updated
+
+    - name: Reboot if kernel updated
```

Comment:
Also note that this trick doesn't play well if the control node is in `all` and gets its kernel updated, because the reboot will interrupt Ansible.  The workaround is to make sure that the kernel is up to date on the control node before you run the playbook against it, or to leave the control node out of `all`.  I ended up adding this because kernel-devel wasn't happy if the kernel got updated but we were still running the old kernel when we tried to do the builds.